### PR TITLE
feat(ProgressBar): allow reverse animation

### PR DESCRIPTION
**Pull request by BrunoGuimaraes103**
Addition of inverted animation option in the progress-bar component, according to Figma:
https://www.figma.com/file/4INjJeoqC7elguwiwg6PqH/Progress-Bar?node-id=0-1&t=fbTFrVoBerKk1vjK-0

### CODE REVIEW

**Comment by atabel** [comment]
Are this changes already in the figma Design specs, @yceballost?

**Comment by AlexJCelmer** [comment]
Hello, follow the discussion link with the approval of moving the component to development

https://github.com/Telefonica/mistica-design/issues/1116

**Comment by yceballost** [comment]
> Are this changes already in the figma Design specs, @yceballost? I see the linked figma is just a proposal

This change is approved by mistica design team, made by vivo design team and included in official mística progrss bar specs.

It is important to work on this evolution using the official specs document to avoid problems in reviews. Vivo design team has to avoid the creation of this parallel specs, or avoid to share with development team.

This is the source of truth for this component
https://www.figma.com/file/3IO1EsGDeBRiFdF23mrvuj/%F0%9F%94%B8-ProgressBar-Specs?node-id=0%3A1&t=eYIOKRVqERQzStn0-1

File: src/progress-bar.tsx [hunk 1]
```diff
@@ -21,6 +23,7 @@ const ProgressBar: React.FC<Props> = ({
     'aria-label': ariaLabel,
     'aria-labelledby': ariaLabelledBy,
     dataAttributes,
+    inverse = false,
```
Comment (yceballost) [inline src/progress-bar.tsx:26]: Don't use 'inverse' for this prop, this name cause conflicts between `inverse` prop in other component that shows inverse state (for color backgrounds)

alternatives...
- inverted?
- animationInverted?

@atabel wdyt?

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/progress-bar.tsx [hunk 2]
```diff
@@ -21,6 +23,7 @@ const ProgressBar: React.FC<Props> = ({
     'aria-label': ariaLabel,
     'aria-labelledby': ariaLabelledBy,
     dataAttributes,
+    inverted = false,
```
Comment (atabel) [inline src/progress-bar.tsx:26]: rename to `reverse` please

**Review by atabel — APPROVED** [review]

**Pull request merged**